.github/workflows/ci.yml

```diff
@@ -10,38 +10,62 @@ on:
     - cron:  '0 12 * * *'
 
 jobs:
-  linux:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: lint
+        run: make lint
+
+  winchecksec:
     strategy:
       matrix:
         build-type: ["Debug", "Release"]
         platform: ["ubuntu-latest", "macos-latest"]
-    runs-on: ubuntu-latest
+    runs-on: ${{ matrix.platform }}
     steps:
       - uses: actions/checkout@v2
+      - name: Restore from cache and install vcpkg
+        uses: lukka/run-vcpkg@v2
         with:
-          submodules: true
-      - name: lint
-        run: make lint
+          vcpkgGitCommitId: 422fb5df87f6ae2d01a6ccdca7211a54c0e88fa1
+          vcpkgArguments: pe-parse
```

Comment:
We might want to use variables for these. It should be possible to add it at the workflow level https://help.github.com/en/actions/reference/workflow-syntax-for-github-actions#env

and an example https://github.community/t5/GitHub-Actions/How-to-set-and-access-a-Workflow-variable/m-p/53516/highlight/true#M8860

Reply:
Good call! And thanks for the example.